packages/terra-list/src/List.jsx

```diff
@@ -17,6 +17,16 @@ const propTypes = {
    * One of `'none'`, `'standard'`, `'bottom-only'`.
    */
   dividerStyle: PropTypes.oneOf(['none', 'standard', 'bottom-only']),
+  /**
+   * The visible or hidden text element on the page that conveys
+   * information about the list and how to interact with it.
+   */
+  instructions: PropTypes.string,
+  /**
+   * String that provides information about the list and how to interact
+   * with it.
+   */
+  instructionText: PropTypes.string,
```

Comment:
What is the difference between these two props?

Reply:
`instructions` references another element on the page that describes the list, `instructionText` is a text string describing the list. As I said [here](https://github.com/cerner/terra-core/pull/3688#discussion_r1041061287), I wonder if it would make more  sense to simply call the props `ariaDescribedBy` and `ariaDescription` to avoid confusion.